docs/manual/developer_guide.adoc

```diff
@@ -555,8 +555,48 @@ Contributions can be made for rules, checks, remediations or even utilities. The
 
 ==== Rules
 
-...
-// shared/references/cce-rhel-avail.txt
```

Comment:
I think this comment is there to make sure the user knows where available CCEs are. It would make sense to describe it. When adding a new rule for RHEL you should assign a CCE ID to it, the file referenced has CCEs allocated for us that haven't been used in SSG yet.